### PR TITLE
added i4i-finance

### PR DESCRIPTION
**Pull request by pureicis**
##### methodology (what is being counted as tvl, how is tvl being calculated): tvl is calculated using the total value of protocol's liquidity pool. Staked tokens include staked EYE values. Pool2 includes staked lp tokens eligible for EYE emissions

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/i4i-finance: 
        
 
  
    
    
    Error: Invalid JSON RPC response: ""
    Truncated error stack:
        at Object.InvalidResponse (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/caver-js/packages/caver-core-helpers/src/errors.js:92:16)
        at ClientRequest.emit (node:events:526:28)
        at TLSSocket.socketOnEnd (node:_http_client:466:9)
        at TLSSocket.emit (node:events:538:35)

File: projects/i4i-finance/index.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  const caver = new Caver(KOKOA_EP_URL);
+  const Helper = new caver.contract(HELPER_ABI.abi, HELPER_ABI.address);
+  const Registry = new caver.contract(REGISTRY_ABI, REGISTRY_ADDR);
+  const decimal = 18;
+  const poolList = await Registry.methods.getPoolList().call();
+
+  let total = BigNumber(0);
+  for (const pool of poolList){
+    const poolVolume = await calcPoolLiquidityVolume(Helper, pool, decimal);
+    total = total.plus(await calcPoolLiquidityVolume(Helper, pool, decimal));
```
Comment (waynebruce0x) [inline projects/i4i-finance/index.js:33]: how come we count twice?

**Comment by waynebruce0x** [comment]
Also see the invalid JSON response above

**Comment by llamatester** [comment]
The adapter at projects/i4i-finance exports TVL: 
        
 
 
    klaytn-staking            1.40 M
    klaytn                    20.80 M
    staking                   1.40 M
    
    total                    20.80 M

**Comment by llamatester** [comment]
The adapter at projects/i4i-finance exports TVL: 
        
 
 
    klaytn-staking            1.40 M
    klaytn                    20.81 M
    staking                   1.40 M
    
    total                    20.81 M

**Pull request merged**